src/osd/scrubber/scrub_job.h

```diff
@@ -178,4 +231,19 @@ struct formatter<Scrub::ScrubJob> {
 	sjob.state.load(std::memory_order_relaxed));
   }
 };
+
+template <>
+struct formatter<Scrub::sched_conf_t> {
+  constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+  template <typename FormatContext>
+  auto format(const Scrub::sched_conf_t& cf, FormatContext& ctx)
+  {
+    return format_to(
```

Comment:
i've opened https://github.com/ceph/ceph/pull/54529 to fix a ftbfs here on gcc13